src/models/events/interaction.rs

```diff
@@ -40,6 +42,19 @@ pub struct SlackInteractionBlockActionsEvent {
     pub state: Option<SlackActionState>,
 }
 
+#[skip_serializing_none]
+#[derive(Debug, PartialEq, Clone, Serialize, Deserialize, Builder)]
+pub struct SlackInteractionBlockSuggestionEvent {
+    pub team: SlackBasicTeamInfo,
+    pub user: SlackBasicUserInfo,
+    pub api_app_id: SlackAppId,
+    pub block_id: SlackBlockId,
+    pub action_id: SlackActionId,
+    pub container: SlackInteractionActionContainer,
+    pub view: Option<SlackView>,
```

Comment:
Nit: the doc says, there is also possibility to see a message optional. Maybe it is also useful to add?

> pub message: Option<SlackHistoryMessage>,

Reply:
Sure, tested and `Option<SlackHistoryMessage>` works well 👍